packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -6487,8 +6567,7 @@ describe('plugin-meetings', () => {
           it('handles "Disconnected" state correctly when waitForIceReconnect resolves', async () => {
             meeting.reconnectionManager.waitForIceReconnect = sinon.stub().resolves();
 
-            mockDisconnectedEvent();
-
+            await mockDisconnectedEvent();
```

Comment:
mockDisconnectedEvent doesn't seem to be an async function right? How does this await help?

Reply:
Yes, this is not required. removed